includes/agentic-commerce/class-wc-stripe-agentic-shipping-package-builder.php

```
@@ -0,0 +1,179 @@
+
+			if ( ! $product_id ) {
+				throw new Exception(
+					sprintf(
+						'Shipping package builder: product not found for line item %s with sku_id "%s".',
+						$line_item->get_id(),
+						$line_item->get_sku_id()
+					)
+				);
+			}
+
+			$product = WC_Stripe_Agentic_Commerce_Product_Resolver::resolve_product( $product_id );
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Agentic_Commerce_Product_Resolver' in method 'build_contents_from_event'. (undefined)

(StaticAccess)